interfaces/builtin/serial_port.go

```diff
@@ -114,8 +114,8 @@ func (iface *serialPortInterface) SanitizeSlot(slot *interfaces.Slot) error {
 		}
 
 		usbInterfaceNumber, ok := slot.Attrs["usb-interface-number"].(int64)
-		if ok && (usbInterfaceNumber < 0x0 || usbInterfaceNumber >= UsbMaxInterfaces) {
-			return fmt.Errorf("serial-port usb-interface-number attribute cannot be negative and larger than 31")
+		if ok && (usbInterfaceNumber < 0 || usbInterfaceNumber >= UsbMaxInterfaces) {
+			return fmt.Errorf("serial-port usb-interface-number attribute cannot be negative and larger than %d", UsbMaxInterfaces-1)
```

Comment:
I think you want to say `... or larger than...`

Reply:
Thanks for the correction. Fixed it.